### ISSUE TITLE
Training on TUD crossing dataset

### ISSUE THREAD

**Issue by CharlesShang** [Russell91/TensorBox#19]
Hi, Russell,
I'm really interested in how to training reinspect on TUD crossing dataset.
I think the major problems come from the insufficient training data. There are only 400 images for training. 
Would you mind sharing some details on TUD crossing experiments?

**Comment by Russell91** [comment]
No, there should be around 2000 images for training and 200 images for testing. TUD crossing is a subset of the 13 piece TUD dataset and is held out for testing.

**Comment by CharlesShang** [comment]
There are several datasets under the name "TUD-**", I'm very confused. 
According to "around 2000 images", I think it's TUD-MotionPairs. But the annotations are not complete, there are many unlabeled people in the training set. Did you have all the people re-labeled? If so, would you mind sharing your annotations? I'll be very grateful!

**Comment by Russell91** [comment]
Yea, Mykhaylo had a turker go through and do a very thorough annotation of the dataset. I'll ask him if he can provide a link.

**Comment by CharlesShang** [comment]
That will be great

**Comment by Russell91** [comment]
Okay, here are the IDL files:

Training:

    http://russellsstewart.com/s/tud/train.idl

Testing:

    http://russellsstewart.com/s/tud/test_hard.idl
    http://russellsstewart.com/s/tud/test_easy.idl

**Comment by CharlesShang** [comment]
Thank you :+1:

**Comment by ghotson** [comment]
Hi Russell,

I'm having some difficulty tracking down where to download some of the files listed in the idl files you posted. I found the TUD Motion Pairs files [here](https://www.mpi-inf.mpg.de/departments/computer-vision-and-multimodal-computing/research/people-detection-pose-estimation-and-tracking/multi-cue-onboard-pedestrian-detection/), and some of the DaSide0811 seq6 and seq7 files [here](https://www.mpi-inf.mpg.de/departments/computer-vision-and-multimodal-computing/research/people-detection-pose-estimation-and-tracking/people-tracking-by-detection-and-people-detection-by-tracking/), but I'm not sure where to find the rest of the images listed in the idl files. Are they publicly available somewhere?

Thanks!

**Comment by gregary** [comment]
I have the same problem with you @ghotson .Did you solve it?
Can you give me the dataset links ? @CharlesShang  @Russell91

**Comment by benslaney** [comment]
are you guys talking about the data set for training? it's right there in "download_data.sh" in the root of the project folder. couldn't be more obvious.

**Comment by Russell91** [comment]
Yea, so I just got this directly from Mykhaylo. It should be downloadable on this page: https://www.mpi-inf.mpg.de/departments/computer-vision-and-multimodal-computing/software-and-datasets/. If it's not in any of those, please send a direct message to Mykhaylo and then post the results here.

**Comment by YangBain** [comment]
@CharlesShang 
Hi CharlesShang,
Recently, I run this program and want to train using TUD crossing datasets. But I cannot find all datasets of DaSide0811. Do you have all seq data of DaSide0811. If you have, whether you can give me a link? Thanks for your help.
Bin Yang
